docs/rest_api.rst

```diff
@@ -111,6 +111,65 @@ The ``GET /patroni`` is used by Patroni during the leader race. It also could be
       }
     }
 
+
+Retrieve the Patroni metrics Prometheus format through the ``GET /metrics`` endpoint.
+
+.. code-block:: bash
```

Comment:
```suggestion
.. code-block:: bash

```

code-block is [not visible](https://github.com/zalando/patroni/blob/f6de2bd0ef2a0ad5f543a10a13e899493e9877dd/docs/rest_api.rst) now :(